unittests/testattachments.py

```diff
@@ -388,44 +369,3 @@ def test_attachments_info(self):
                 INFO,
                 msg="INFO method called incorrectly",
             )
-
-    def test_attachments_info_with_asset_id(self):
```

Comment:
Although they're not identical I assume all this removed coverage is replaced by the much more comprehensive dedicated attachment tests

Reply:
yes - that is the intention